stories/tokens.stories.js

```diff
@@ -0,0 +1,345 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Cell, Grid, Row } from '../assets/js/material-components';
+import { Fragment } from '@wordpress/element';
+
+import './tokens.scss';
+
+function TokenRow( { children } ) {
+	return <div className="googlesitekit-color-token-row">{ children }</div>;
+}
+function TokenColumn( { children } ) {
+	return <div className="googlesitekit-color-token-column">{ children }</div>;
+}
+
+function Token( { name, label, reference } ) {
+	return (
+		<div
+			className={ `googlesitekit-color-token googlesitekit-color-token--${ name }` }
+		>
+			<p className="googlesitekit-color-token--label">{ label }</p>
+			<p className="googlesitekit-color-token--reference">
+				{ reference }
+			</p>
+		</div>
+	);
+}
+
+const SurfacesTokens = () => (
+	<Fragment>
+		<h2>Surfaces</h2>
+		<TokenRow>
+			<Token
+				name="surfaces-on-background"
+				label="On Background"
+				reference="N-900"
+			/>
+			<Token
+				name="surfaces-on-background-variant"
+				label="On Background Variant"
+				reference="N-500"
+			/>
+		</TokenRow>
+		<Token name="surfaces-background" label="Background" reference="N-10" />
+		<Token
+			name="surfaces-on-surface"
+			label="On Surface"
+			reference="N-900"
+		/>
+		<Token
+			name="surfaces-on-surface-variant"
+			label="On Surface Variant"
+			reference="N-500"
+		/>
+		<Token name="surfaces-surface" label="Surface (0)" reference="N-0" />
+		<Token name="surfaces-surface-1" label="Surface 1" reference="N-50" />
+		<Token name="surfaces-surface-2" label="Surface 2" reference="N-100" />
+		<TokenRow>
+			<Token
+				name="surfaces-inverse-on-surface"
+				label="Inverse On Surface"
+				reference="N-900"
+			/>
+			<Token
+				name="surfaces-inverse-surface"
+				label="Inverse Surface"
+				reference="N-900"
+			/>
+		</TokenRow>
+	</Fragment>
+);
+
+const ContentTokens = () => (
+	<Fragment>
+		<h2>Content</h2>
+		<TokenRow>
+			<Token
+				name="content-on-primary-container"
+				label="On Primary Container"
+				reference="SK-600"
+			/>
+			<Token
+				name="content-on-primary"
+				label="On Primary"
+				reference="SK-0"
+			/>
+		</TokenRow>
+		<TokenRow>
+			<Token
+				name="content-primary-container"
+				label="Primary Container"
+				reference="SK-100"
+			/>
+			<Token name="content-primary" label="Primary" reference="SK-600" />
+		</TokenRow>
+		<TokenRow>
+			<Token
+				name="content-on-secondary-container"
+				label="On Secondary Container"
+				reference="T-600"
+			/>
+			<Token
+				name="content-on-secondary"
+				label="On Secondary"
+				reference="T-0"
+			/>
+		</TokenRow>
+		<TokenRow>
+			<Token
+				name="content-secondary-container"
+				label="Secondary Container"
+				reference="T-50"
+			/>
+			<Token
+				name="content-secondary"
+				label="Secondary"
+				reference="SK-600"
```

Comment:
Similarly, as you've flagged on Figma, it looks like this should be `T-500`. We need @sebastianmantel to confirm but I suspect this will need to be changed.

```suggestion
				reference="T-500"
```

Reply:
Updated.